DcCore/DcCore/DC/events.swift

```diff
@@ -218,10 +219,10 @@ public class DcEventHandler {
             if let sem = dcAccounts.fetchSemaphore, dcAccounts.isAllWorkDone() {
                 sem.signal()
             }
-            if dcContext.id != dcAccounts.getSelected().id {
+            if accountId != dcAccounts.getSelected().id {
                 return
             }
-            dcContext.logger?.info("network: DC_EVENT_CONNECTIVITY_CHANGED: \(dcContext.getConnectivity())")
+            dcContext.logger?.info("network: DC_EVENT_CONNECTIVITY_CHANGED: \(self.dcAccounts.get(id: accountId).getConnectivity())")
```

Comment:
The event does not contain the current state so this function call keeps being necessary.